### ISSUE TITLE
deck 1.28 introduces breaking changes

### ISSUE THREAD

**Issue by rspurgeon** [Kong/deck#1071]
- Commands marked as deprecated issue the deprecated warning to STDOUT causing a breaking change for any successful expected output.  For example:
```
deck dump --format json --yes | jq .
jq: parse error: Invalid numeric literal at line 1, column 8
```
- deck dump writes to STDOUT by default where prior it wrote to kong.<extension> by default
- deck gateway sync hangs and cannot be broken by Ctrl-C on the CLI process

**Comment by Tieske** [comment]
item 2: see https://github.com/Kong/deck/pull/1073 for the fix

**Comment by Tieske** [comment]
item 3 tracked in #1077 , others fixed in 1.28.1, closing this.
